src/nile/deployments.py

```diff
@@ -28,6 +28,57 @@ def register(address, abi, network, alias):
         fp.write("\n")
 
 
+def update_abi(address_or_alias, abi, network):
+    """
+    Update the ABI for an existing deployment that matches an identifier.
+
+    If address_or_alias is an int, address is assumed.
+
+    If address_or_alias is a str, alias is assumed.
+    """
+    file = f"{network}.{DEPLOYMENTS_FILENAME}"
+
+    if not os.path.exists(file):
+        raise Exception(f"{file} does not exist")
+
+    with open(file, "r") as fp:
+        lines = fp.readlines()
+
+    found = False
+    for i in range(len(lines)):
+        line = lines[i].strip().split(":")
```

Comment:
If there are multiple aliases, this method removes them except for the first one. I don't know if this behavior is documented, but other methods (like load) assume that multiple aliases can exist (getting all of them in *aliases) from the line. I suggest refactoring this to allow multiple aliases separated with a colon (keep them and recognize them).

Reply:
Changed to handle multiple aliases.

How should multiple aliases be added in the first place though?  (For the tests, I'm assuming they are registered with a string like "alias1:alias2", since the `register` command does not currently allow appending an alias to an existing deployment)

Reply:
Issue #220 should add better specifications for this. For this PR, I think maintaining consistency with the existing codebase is enough.